pkg/features/embedded/elemental-setup/etc/dracut.conf.d/02-elemental-setup.conf

```diff
@@ -1,4 +1,2 @@
-install_items+=" /usr/lib/systemd/system/elemental-setup-initramfs.service /etc/systemd/system/initrd.target.requires/elemental-setup-initramfs.service "
-install_items+=" /usr/lib/systemd/system/elemental-setup-rootfs.service /etc/systemd/system/initrd-fs.target.requires/elemental-setup-rootfs.service "
```

Comment:
I saw dracut error messages trying to execute those, that how all this PR started.